src/integ_test_resources/android/amplify/integration/cdk/scripts/setup_amplify

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
If you exclude `.py` from the file name, and if you intend for this to be directly executed on the command line (`./setup_amplify`) it'd be good to include a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the start of the document:
```shell
#!/usr/bin/env python3
```
or similar.